jetbrains-rider/src/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotNetLambdaHandlerResolver.kt

```diff
@@ -100,6 +86,39 @@ class DotNetLambdaHandlerResolver : LambdaHandlerResolver {
         return isMethodExists
     }
 
+    /**
+     * Disable frontend line markers since Rider get them from backend.
+     * Please see [software.aws.toolkits.jetbrains.services.lambda.LambdaHost]
+     */
+    override fun shouldShowLineMarker(handler: String): Boolean = false
+
+    fun getFieldIdByHandlerName(project: Project, handler: String): Int {
```

Comment:
Let's pass all the needed information from the backend as a struct containng `className , methodName, targetFramework, projectId` instead of `string`?

Reply:
Is this API needed in the first place?